homeassistant/components/sensor/__init__.py

```diff
@@ -21,10 +22,18 @@
     DEVICE_CLASS_HUMIDITY,
     DEVICE_CLASS_ILLUMINANCE,
     DEVICE_CLASS_MONETARY,
+    DEVICE_CLASS_NITROGEN_DIOXIDE,
+    DEVICE_CLASS_NITROGEN_MONOXIDE,
+    DEVICE_CLASS_NITROUS_OXIDE,
+    DEVICE_CLASS_OZONE,
+    DEVICE_CLASS_PM1,
+    DEVICE_CLASS_PM10,
+    DEVICE_CLASS_PM25,
```

Comment:
I would suggest calling PM25 something like PM2_5 or similar - to indicate that it is not 25 in size?

Reply:
If memory serves I did it like this because there's already some precedent for PM25 in code base.

I've already done the 3 ancillary PRs so I really don't want to change this unless core strongly objects or someone wants to chip in to all 4 PRs?

Reply:
Is there a standard measurement that is 25 in size that this could be confused with?

Reply:
No, there is nothing else than 2.5 I would say. My comment is just for consistency - but anyone that know PM types will be able to guess that it is 2.5 rather than 25.

Reply:
No, in fact if you google pm25 the results make it clear they are about 2.5.

Reply:
Here is one of the existing uses. I'd like to switch it to use the actual const if the PR is accepted and ditch the component local one:

https://github.com/home-assistant/core/blob/df03cce471cb6368bd0d52ce95a4f13a97cb0bea/homeassistant/components/homekit/const.py#L244

Which i can safely do if we are compatible with it.

Reply:
Ok, let's keep it like this then.